blog.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adopt a Dog Blog</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+    <header class="header">
+      <img class="header-img" src="images/adoptadog-logo.svg"  alt="Logo">
+      <h1 class="header-text">Seattle's premier dog adoption agency</h1>
+    </header>
+  <section class="navbar">
+    <nav>
+      <div class="nav">
+      <a class="header-links" href="index.html">Home</a>
```

Comment:
these anchor tags need to be indented in one level from the parent. If you delete the unused div, they'll be fine, but if you keep it, you'll need to indent these